test/ibmq/runtime/test_runtime_integration.py

```diff
@@ -154,6 +155,32 @@ def test_upload_program(self):
         self.assertTrue(program)
         self.assertEqual(max_execution_time, program.max_execution_time)
 
+    def test_update_program(self):
+        """Test updating a program.
+        NOTE: When an Qiskit Runtime API endpoint is created to GET
+         a runtime program in plaintext, update this test to verify the
+         program is modified.
+        """
+        # Create some program
+        program_id = self._upload_program()
+        self.assertTrue(program_id)
+        # Update the program data (last 2 lines differ from original)
+        new_program = str(self.RUNTIME_PROGRAM)
+        new_program.replace('warnings.warn("this is a stderr message")',
+                            'warnings.warn("this is not a stderr message")')
+        # Execute with bytes
+        self.provider.runtime.update_program(program_id, new_program.encode())
+        # Prepare file data
+        pfile = tempfile.NamedTemporaryFile(mode="w+")
```

Comment:
Could you split this into a separate test?

Reply:
[Updated!](https://github.com/Qiskit/qiskit-ibmq-provider/pull/1008/commits/58a4d730cee9f0eddd6eec34628def2da27cf62c#diff-68d8af3bb045bfa53f3f7c127891b2276392f8797f0d3a768c151f6a2f41869cR159-R185)